src/rgw/rgw_metadata.cc

```diff
@@ -291,6 +291,29 @@ RGWCoroutine* RGWMetadataLog::purge_cr()
 			      std::move(oids));
 }
 
+RGWCoroutine* RGWMetadataLog::peer_trim_cr(int shard_id, std::string marker)
+{
+  // Extract timestamp from marker
+  auto oid = get_shard_oid(shard_id);
+
+  ceph::real_time stable;
+  int sec, usec;
+  char keyext[256];
+
+  int ret = sscanf(marker.c_str(), "1_%d.%d_%255s", &sec, &usec, keyext);
+
+  if (ret) {
```

Comment:
This doesn't look right? At least in my manpage for sscanf it says:

```
On success, these functions return the number of input items successfully matched and assigned; this can be fewer  than  provided  for,  or even zero, in the event of an early matching failure.
```

Reply:
oops..thanks. corrected it.